librz/include/rz_util/rz_assert.h

```diff
@@ -137,19 +137,19 @@ RZ_API void rz_assert_log(RzLogLevel level, const char *fmt, ...) RZ_PRINTF_CHEC
 
 #define rz_return_if_fail(expr) \
 	do { \
-		assert(expr); \
+		rz_return_if_fail(expr); \
```

Comment:
In this file this change is unnecessary - it will just make it recursive

Reply:
Okay so I should still use assert instead?

Reply:
In this file, which defines macroses like `rz_return_if_fail()` - yes.